firmware_mod/www/cgi-bin/action.cgi

```diff
@@ -742,6 +742,26 @@ if [ -n "$F_cmd" ]; then
         fi
         return
         ;;
+     recording_files)
+      for file in $(find /system/sdcard/DCIM/Recording/ -type f)
+        do
+         if [[ -f $file ]]; then
+          file_size=$(ls -lh $file | awk '{print $5}')
+          file_url=$(ls -lh $file | awk '{print $9}' | sed 's/\/system\/sdcard\/DCIM/https:\/\/192.168.76.105\/viewer/')
```

Comment:
ip adress should be more generic :)
you could use this to get your cameras ip adress`ip -o -4 addr show dev eth0 | sed 's/.* inet \([^/]*\).*/\1/'`

Reply:
well `ip -o -4 addr show dev wlan0 | sed 's/.* inet \([^/]*\).*/\1/'`

Reply:
not sure what to do about them usb ethernet users

Reply:
I will correct thoses mistakes...